angularjs-portal-home/src/main/webapp/js/app-config.js

```diff
@@ -92,6 +91,12 @@ define(['angular'], function(angular) {
             "title" : "Old MyUW"
           },
         ])
+        .value('SEARCH_URLS', [
```

Comment:
Maybe name this `DIRECTORY_SEARCH_CONFIG` or something like that?

Could also change the structure to be 

```json
[
 {
  "group" : "UW-Madison",
  "url" : "/web/api/wiscdirectory"
 }
]
```

This way we could possibly extend a directory search to other campuses.